### PR TITLE
DESKTOP-version pull

### PR DESCRIPTION
**Pull request by Ramaunspoken**
I've edited and updated the css and html for the Desktop version of the portfolio.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,225 +1,320 @@
...
-            <div class="header_logo blue"><a class="blue" href="#">My Logo</a></div>
-            <div class="header_menu">
-                <img class="menu_icon" src="./images/Union.png" alt="menu" />
-            </div>
-        </nav>
+      <nav class="header_nav">
+        <div class="header_logo blue"><a class="blue" href="#">My Logo</a></div>
+        <div class="header_menu">
+          <img class="menu_icon" src="./images/Union.png" alt="menu" />
+        </div>
```
Comment (Qoosim) [inline index.html]: - [ ] Greate job for  making it this far, I must commend your effort for that. :handshake: . However, on the desktop version of your project, you're supposed to display the menu items at the top right corner of your page as we have it on the Figma template you're mimicking. Those navigation items are not present on your page. Kindly look at the both designs, find out the differences and work your project in line with the Figma design. Thank you.

### Your Design

your-menu-items

### Figma Design

figma-menu-items

File: index.html [hunk 2]
```diff
@@ -1,225 +1,320 @@
...
+              sign-ups required.
+            </p>
+            <ul class="work_cats">
+              <li>Html</li>
+              <li>Css</li>
+              <li>Javascript</li>
+            </ul>
+            <button type="button" class="default_btn">See More</button>
+          </div>
+        </div>
```
Comment (Qoosim) [inline index.html]: - [ ] Looking at the design on the Figma template. The card is divided into two equal parts. One for the image and the other one for the image info. In your own case, the info part takes a larger part of the card which is not in line with what we have on the guiding template. Kindly make your project to produce a close replica of what we have on the guiding template. This applies to all other work-cards in your project. Take a look at the both designs and work your design along side with the guiding template.

### Your Design

your-project-card

### Figma Design

figma-project-card

File: index.html [hunk 3]
```diff
@@ -1,225 +1,320 @@
...
+            Skills
+            <img
+              class="arrow_icon"
+              src="./images/Icon - Chevron - Right - Larger.png"
+              alt="down"
+            />
+          </h3>
+        </ul>
+        </div>
+      </section>
```
Comment (Qoosim) [inline index.html]: - [ ] You also need to work on this section of your project in order to produce the exact design of what we have on the Figma template. Consider giving your about section margin or padding to the both sides and give more width to `languages` side than `about me` side. Take a look at the both designs and fix your project to produce what is on the Figma template.

#### Your Design

your-about-section

### figma Design
figma-about-section

File: index.html [hunk 4]
```diff
@@ -1,225 +1,320 @@
...
+            <textarea
+              class="form_text_area"
+              name="message"
+              placeholder="Write your message here"
+              maxlength="500"
+              required
+            ></textarea>
+            <button type="submit" class="btn_submit">Get It Touch</button>
+          </fieldset>
+        </form>
```
Comment (Qoosim) [inline index.html]: - [ ] Your contact me section also needs some adjustment. The text at the top of your form covers more space compare to what is on the Figma design. kindly take a look at both designs and adjust your design to produce what we have on the guiding template.
- [ ] Kindly take note of the position of the `Get in touch` button on the desktop. It is ought to be aligned to the center of the form. Kindly look into that too and fix it.

### Your Design
your-contact-section

### Figma Design

figma-contact-section

**Review by Qoosim — CHANGES_REQUESTED** [review]
#  *Status: Required Changes* :recycle: 

Hi @Ramaunspoken, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

-  No linter errors. :heavy_check_mark: 
-  Good README file. :heavy_check_mark: 
-   Correct GitHub Flow. :heavy_check_mark: 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Nemwel-Boniface — APPROVED** [review]
## Hello @Ramaunspoken      :wave: ,
I am Nemwel your reviewer for this PR :man_beard:  

Thank you for making the requested changes good job! :partying_face: 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! :partying_face: 🎉 



## Highlights

Great job setting up the correct linters ✔️ 
Professional documentation ✔️ 
Correct workflow ✔️ 
Meaningful commit messages ✔️ 
Met all the milestone requirements ✔️ 

## [Optional] suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me **@Nemwel-Boniface** in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**